exporter/chronicleforwarderexporter/README.md

```diff
@@ -0,0 +1,55 @@
+# Chronicle Forwarder Exporter
+
+The Chronicle Forwarder Exporter is designed for forwarding logs to a Chronicle endpoint using either Syslog or File-based methods. This exporter supports customization of data export types and various configuration options to tailor the connection and data handling to specific needs.
```

Comment:
Does it forward logs to a chronicle endpoint or an existing chronicle forwarder? Maybe this is right the wording just does not match my current understanding.

Reply:
yeah i put `chronicle forwarder endpoint` below this, updated this to match that